insights/core/spec_factory.py

```diff
@@ -705,6 +705,8 @@ def __call__(self, broker):
                     results.append(self.kind(
                         path[len(root):], root=root, save_as=self.save_as,
                         ds=self, ctx=ctx, cleaner=cleaner))
+                except NoFilterException as nfe:
+                    raise nfe
                 except Exception:
                     log.debug(traceback.format_exc())
```

Comment:
Shall we also add this `NoFilterException` for `command_with_args` at  [insights/core/spec_factory.py#L969](https://github.com/RedHatInsights/insights-core/blob/insights-core-3.4.9/insights/core/spec_factory.py#L969) ?

Reply:
Nice catching. Added